documentatie/gebruikersdocumentatie/verkiezingsproces-en-abacus.md

```diff
@@ -0,0 +1,165 @@
+helpt om kleine fouten in een vroeg stadium op te sporen.
+
+## De rol van Abacus
+
+Het doel van Abacus is om de papieren processen-verbaal te digitaliseren en te helpen met de controle op de optellingen, zodat we tel- en schrijffouten kunnen detecteren en voorkomen. De software wordt ontwikkeld ter vervanging van het onderdeel Uitslagvaststelling (U) van OSV2020. Hierbij hebben we een aantal doelen gesteld:
+
+- De integriteit, transparantie en controleerbaarheid van het gehele proces ondersteunen.
+- Gebruiksvriendelijke interface en technologie implementeren.
+- Volledig open source ontwikkelen.
+- De oplossing zelf ontwikkelen en beheren.
+
+Abacus wordt in de eerste instantie ontwikkeld voor de Gemeenteraadsverkiezingen in maart 2026. Meer informatie over de functionaliteit die we gaan bouwen vind je in [Functionaliteit voor Abacus 1.0](/documentatie/functionaliteit/versie-1.0-gr2026.md).
+
+### Gebruiksvriendelijkheid
+
+Omdat het verkiezingsproces snel verloopt en er niet al te vaak verkiezingen zijn, maken de gebruikers slechts korte tijd gebruik van verkiezingssoftware. Daarom is het van groot belang dat Abacus zo gebruiksvriendelijk mogelijk is. Hiervoor implementeren we de volgende maatregelen:
+
+- We doen regelmatige gebruikerstests met echte gebruikers.
+- De invoer wordt niet allemaal op één pagina gedaan zoals in de huidige software, maar in kleinere secties met een validatiecheck na elke invoer.
+- Je kunt de invoer pauzeren en later doorgaan.
+- De software wordt geoptimaliseerd voor gebruik met een toetsenbord en is eenvoudig aan te leren.
+- De software geeft duidelijk aan hoe onjuiste of inconsistente invoer kan worden opgelost.
+- We bieden duidelijke documentatie en instructiemateriaal.
+
+Dit is een voorbeeld van de gebruikersinterface:
+
+![Interface](/documentatie/gebruikersdocumentatie/img/abacus-interface.png)
+
+### Functionaliteit
+
+In de afbeelding hieronder zie je hoe Abacus werkt op het gemeentelijk stembureau. Voor landelijke verkiezingen wordt dit proces herhaald op het hoofdstembureau en centraal stembureau, zoals hierboven beschreven.
+
+**LET OP: dit is een concept en is nog in ontwikkeling.**
+
+![Uitslagvaststelling](/documentatie/gebruikersdocumentatie/img/proces%20uitslagvaststelling.png)
+
+- IMPORT: gebruikers kunnen kandidaten en verkiezingsbestanden in EML-formaat importeren in Abacus.
+- INVOER: gebruikers kunnen verkiezingsresultaten invoeren, inclusief verklaringen over ontbrekende stembiljetten/stempassen en klachten van kiezers.
+Abacus telt de resultaten op en controleert ze.
+De data-analyse leidt tot waarschuwingen voor kleine afwijkingen en foutmeldingen voor grote afwijkingen en inconsistente resultaten.
+- ONDERTEKENING: de data wordt ondertekend met een handtekening van de voorzitter van het gemeentelijk stembureau.
+- EXPORT: de verkiezingsresultaten worden geëxporteerd als EML_NL-bestand en als PDF-bestand.
```

Comment:
Ik moest twee keer kijken voordat ik doorhad dat "IMPORT" etc naar de blauwe kaders rond het Abacus-logo verwijzen. (Had ook niet lang naar de afbeelding gekeken, want "Oh ja, die ken ik.")

Reply:
Ik heb even toegevoegd waar de woorden in het plaatje staan.